src/panels/Scene/SceneGraphNode/SceneGraphNodeMetaInfo.tsx

```diff
@@ -47,49 +49,61 @@ export function SceneGraphNodeMetaInfo({ uri }: Props) {
       [
         'Identifier:',
         <Group justify={'space-between'}>
-          {identifier}
+          <Text className={classes.selectable}>{identifier}</Text>
           <CopyToClipboardButton value={identifier || ''} />
         </Group>
       ],
       [
         'URI:',
         <Group justify={'space-between'}>
-          <Code>{uri}</Code>
+          <Code className={classes.selectable}>{uri}</Code>
           <CopyToClipboardButton value={uri} />
         </Group>
       ],
-      ['About:', description || 'No description found'],
+      [
+        'About:',
+        <Text className={classes.selectable}>
+          {description || 'No description found'}
+        </Text>
+      ],
       [
         'Tags:',
         <Group gap={'xs'}>
           {propertyOwner?.tags.map((tag) => (
             <Pill key={tag}>
               <Flex gap={2}>
-                {tag}
+                <Text className={classes.selectable}>{tag}</Text>
```

Comment:
```suggestion
                <span className={classes.selectable}>{tag}</span>
```
Making this a text component borked up the size, and there is no need for this to be a text